src/services/component-service.ts

```diff
@@ -133,6 +134,29 @@ export class ComponentService {
     }
   }
 
+  public checkConfigMfes(): void {
+    const allMfes = this.getComponents(ComponentType.MICROFRONTEND)
+    const allMfesWithTypes = allMfes.map(({ name, others }) => ({
+      name,
+      type: others?.type
+    }))
+    for (const mfe of allMfes) {
+      if (
+        mfe.others?.configMfe &&
+        mfe.others.configMfe.slice(0, 9) !== 'internal:' &&
+        allMfesWithTypes.findIndex(
+          ({ name, type }) =>
+            mfe.others.configMfe === name &&
+            type === MicroFrontendType.WidgetConfig
+        ) === -1
+      ) {
+        throw new CLIError(
+          `Config MFE ${mfe.others.configMfe} must be at least an internal widget or must exist among your micro frontends with type widget-config`
```

Comment:
configMfe value ${mfe.others.configMfe} for MFE ${mfe} must be an existing widget-config MFE in the project or a platform provided config MFE, defined with the internal:<name> syntax

Reply:
done